module/classes/DevModeConfig.mjs

```diff
@@ -188,9 +192,12 @@ export class DevModeConfig extends FormApplication {
 
         switch (event.currentTarget?.dataset?.action) {
           case 'actorCRUD': {
-            const inputVal = html.find('input[name="actorCrudIterations"]').val();
-            console.log('inputVal', inputVal);
-            return DevModePerformance.actorCRUDTest(inputVal || undefined);
+            const form  = new FormDataExtended(event.currentTarget.closest('form')).toObject();
+            const actorType = form.actorCrudType;
+            const iterations = form.actorCrudIterations;
+            const synthetic = form.actorCrudSynthetic;
+            console.log('CRUD', { actorType, iterations, synthetic });
+            return DevModePerformance.actorCRUDTest(actorType, iterations || undefined, synthetic);
```

Comment:
If we use an object as the param for `actorCRUDTest`, we might be able to pass the output of `formData` directly into this method.

Reply:
Giving form data directly would give really ugly parameter names, unless the CRUD test is given its own form. That gets into rather large redesigns of the settings dialog.

Reply:
I think I had an idea for a solution.